docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing guidelines
+
+The rtl_433 project accepts contributions.
+Please check if your contribution is following these guidelines
+to improve the feedback loop and decrease the burden for the maintainers.
+
+## Commit messages
+
+The commit messages should follow the described format.
+
+<area_of_work>: <verb> <commit_message>
+
+Area of work may be one of the following:
+
+- build: for build/build system related work
+- docs: for documentation related work, both in code and readme/docs folder
+- ci: for work related to continuous integration
+- test: for test related work 
+- <decoder_name>: for decoder related work
+- decoders: when multiple decoders are changed at once
+- deps: for changes related to (external) dependencies (e.g. soapysdr is updated or mongoose is updated)
+- core: for changes related to the core decoding and parsing logic (e.g. bitbuffer parsing, fsk decoding)
+- minor: for housekeeping work, code style changes
+
+Verb may be one of the following:
+
+- Add: for new additions, e.g. device support
+- Fix: for changes that don't change anything to input/output (security related or bug fixing)
+- Remove: for changes that remove behaviour (e.g. some old algorithms are cleaned up)
+- Change: for changes that modify input/output behaviour (e.g. added checksums, preambles)
+- Improve: for improvements without changes in normal output/behaviour
+
+## Supporting Additional Devices and Test Data
+
+Some device protocol decoders are disabled by default. When testing to see if your device
+is decoded by rtl_433, use `-G` to enable all device protocols.
+This will likely produce false positives, use with caution.
+
+The first step in decoding new devices is to record the signals using `-S all`.
```

Comment:
-S unknown?

Reply:
Good catch

Reply:
I think the section might need some more love, but that's out of scope for this PR. I'm just moving it from the readme.